packages/ember-metal/lib/map.js

```diff
@@ -186,7 +186,7 @@ class OrderedSet {
 
   /**
     @method copy
-    @return {Ember.OrderedSet}
+    @return {OrderedSet}
     @private
   */
   copy() {
```

Comment:
on line 220, we still have `@namespace` for Map.  We should remove this line so Map shows up as `Map` in the docs and not `Ember.Map`